specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/VpnSiteLinkList.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
This example file isn't referenced by the specs

Reply:
removed